servicebroker/config/settings.json

```diff
@@ -2,5 +2,6 @@
   "port": 8080,
   "username": "username",
   "password": "password",
-  "dbUri": "postgres://postgres@127.0.0.1:5432/autoscaler"
+  "dbUri": "postgres://postgres@127.0.0.1:5432/servicebroker_db",
```

Comment:
this does not match the README that is available for the app-autoscaler.
while this dbUri is for the servicebroker_db, right now for development, we have 1 DB, autoscaler.